x-pack/test_serverless/api_integration/test_suites/observability/custom_threshold_rule/avg_pct_fired.ts

```diff
@@ -23,22 +23,27 @@ export default function ({ getService }: FtrProviderContext) {
   const dataViewApi = getService('dataViewApi');
   const logger = getService('log');
 
-  // Blocked API: index_not_found_exception: no such index [.alerts-observability.threshold.alerts-default]
-  // Issue: https://github.com/elastic/kibana/issues/165138
-  describe.skip('Custom Threshold rule - AVG - PCT - FIRED', () => {
+  describe('Custom Threshold rule - AVG - PCT - FIRED', () => {
     const CUSTOM_THRESHOLD_RULE_ALERT_INDEX = '.alerts-observability.threshold.alerts-default';
+    // DATE_VIEW should match the index template:
+    // x-pack/packages/kbn-infra-forge/src/data_sources/composable/template.json
+    const DATE_VIEW = 'kbn-data-forge-fake_hosts';
```

Comment:
Can we expose this const from the package and use it both inside the package and in the integration tests instead of defining them multiple times?

Reply:
@maryam-saeidi, this value should not be part of the package as it depends on the test we are performing. e.g., for this rule and this test, we are using the template of fake_host. 
But later, we could use a different type of data, we need to define another template and another DATA_VIEW.

Reply:
Do you mean we will add other metrics besides `fake_host` in the future so the DATA_VIEW will be different? Can't we have multiple DATA_VIEWs exposed, when we add more scenarios?

Would you have any suggestions about improving the current case and not relying on the comment? It can be tricky to debug when the data view in the package is changed, as in the package, we don't have a connection between the logic and what is used in tests.

Reply:
> Do you mean we will add other metrics besides fake_host in the future so the DATA_VIEW will be different? Can we have multiple DATA_VIEWs exposed when we add more scenarios?

We could.

There is always room for improvement, but for the current tests and data, it's totally okay.